x/leverage/simulation/genesis.go

```diff
@@ -20,17 +20,17 @@ const (
 
 // GenCompleteLiquidationThreshold produces a randomized CompleteLiquidationThreshold in the range of [0.050, 0.100]
 func GenCompleteLiquidationThreshold(r *rand.Rand) sdk.Dec {
-	return sdk.NewDecWithPrec(050, 3).Add(sdk.NewDecWithPrec(int64(r.Intn(950)), 3))
+	return sdk.NewDecWithPrec(50, 3).Add(sdk.NewDecWithPrec(int64(r.Intn(950)), 3))
```

Comment:
Note: formatter actually wanted to change this to `0x050` (octal), which is not what the code was intended to do, but it was actually doing.

Manually removed leading zeroes in the three lines changed for this file.